### PR TITLE
Update create-dns-records-at-any-dns-hosting-provider.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note for Teams in the Document

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1759

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/get-help-with-domains/create-dns-records-at-any-dns-hosting-provider.md [hunk 1]
```diff
@@ -169,9 +169,12 @@ On your DNS host's website, you'll create three new CNAME records, typically one
       |CNAME (Alias)  <br/> |sip  <br/> |sipdir.online.lync.com  <br/> |1 hour  <br/> |
    
    > [!NOTE]
-   > For **TTL**: Set this value to **1 hour** or to the equivalent in minutes ( **60** ), seconds ( **3600** ), etc. > These records do not apply to Exchange, Lync, or Skype for Business hybrid deployments. 
+   > For **TTL**: Set this value to **1 hour** or to the equivalent in minutes ( **60** ), seconds ( **3600** ), etc. .These records do not apply to Exchange, Lync, or Skype for Business hybrid deployments.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/create-dns-records-at-any-dns-hosting-provider.md]:
```suggestion
   > For **TTL**: Set this value to **1 hour** or to the equivalent in minutes (**60**), seconds (**3600**), etc. .These records do not apply to Exchange, Lync, or Skype for Business hybrid deployments. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@pebaum - PR has been copy edited and is ready for final review. This is based on issue #1759.

Thank you.

cc: @kenwith

**Comment by pebaum** [comment]
this article is in the process of a major restructure. closing without merge but will consider feedback on new version.

**Pull request closed without merging**